src/test/components/TrackNetwork.test.js

```diff
@@ -57,7 +57,10 @@ describe('timeline/TrackNetwork', function() {
     // Ensure we start out with 0.
     expect(getContextDrawCalls().length).toEqual(0);
 
-    // Send out the resize, and ensure we are drawing.
+    // Send out the resize with a width change, and ensure we are drawing.
+    HTMLElement.prototype.getBoundingClientRect.mockImplementation(() =>
+      getBoundingBox(GRAPH_WIDTH - 100, GRAPH_HEIGHT)
+    );
```

Comment:
Yeah, I believe this is it: I'm forcing a change of the values so that the PureComponent would update. I can double check and put some more precision in the comment.